packages/netlify/src/index.ts

```diff
@@ -332,6 +363,8 @@ export default function netlifyIntegration(
 			get cookies(): never {
 				throw new Error('Please use Astro.cookies instead.');
 			},
+			// @ts-expect-error
```

Comment:
nit: it could be useful to add a comment that explains why we should except an error here